.github/workflows/prod-build.yml

```diff
@@ -273,15 +282,11 @@ jobs:
           REACT_APP_MDN_PLUS_10M_SP3_ID: mdnplus10m
           REACT_APP_MDN_PLUS_10Y_SP3_ID: mdnplus10y
 
-          # Surveys.
-          REACT_APP_SURVEY_START_JS_PROPOSALS_2025: 1743120000000 # new Date("2025-03-28Z").getTime()
-          REACT_APP_SURVEY_END_JS_PROPOSALS_2025: 1744502400000 # new Date("2025-04-13Z").getTime()
-          REACT_APP_SURVEY_RATE_FROM_JS_PROPOSALS_2025: 0.0
-          REACT_APP_SURVEY_RATE_TILL_JS_PROPOSALS_2025: 0.05 # 5%
-
           # Telemetry.
           REACT_APP_GLEAN_CHANNEL: prod
           REACT_APP_GLEAN_ENABLED: true
+          FRED_GLEAN_CHANNEL: prod
+          FRED_GLEAN_ENABLED: true
 
           # Newsletter
           REACT_APP_NEWSLETTER_ENABLED: true
```

Comment:
Is this still needed?

Reply:
I think it's still required to show the deprecation notice